api/v1alpha1/contour_types.go

```diff
@@ -302,6 +317,75 @@ const (
 	GCPLoadBalancerProvider   LoadBalancerProviderType = "GCP"
 )
 
+type AwsLoadBalancerParameters struct {
+	// AllocationID is a list of Allocation IDs of Elastic IP addresses that are
+	// to be assigned to created Network Load Balancer.
+	// Example: "eipalloc-<xxxxxxxxxxxxxxxxx>"
```

Comment:
- Add a blank line between the field description and the example.
- Although the description references "Network Load Balancer", we need to be explicit that the LB type must be NLB. Since LB type is currently not supported, we need to add NLB support before this PR so we can point to the specific LB type field.

Reply:
Fixed